docs/quickstart.md

```diff
@@ -9,7 +9,7 @@ We need some imports to get started:
 
 ```python
 import numpy as np
-import time
+from tinygrad.helpers import Timing
```

Comment:
I totally buy that for a quickstart it might be better to just use time to decrease the amount of new stuff the reader has to deal with.

But it is a nice utility function to know about so hence this suggestion. But I'm not sure myself.

Reply:
Yea, I like this better, it showcases the feature.